app/qml/account/MMWhichIndustryPage.qml

```diff
@@ -12,6 +12,7 @@ import QtQuick.Controls
 
 import mm 1.0 as MM
 
+import "./components"
 import "../components"
```

Comment:
Hmm, just wondering if we should name the subfolders like this... I know we use it in other places too, I am just a bit afraid that you can import "../components" as well as "./components" 😵‍💫